roles/openshift_logging_mux/templates/mux.j2

```diff
@@ -115,6 +123,18 @@ spec:
             resourceFieldRef:
               containerName: "mux"
               resource: limits.memory
+        - name: "BUFFER_TYPE"
+          value: "{{ openshift_logging_mux_buffer_type | lower }}"
+        - name: "FILE_BUFFER_PATH"
+{% if openshift_logging_mux_file_buffer_storage_type == 'pvc' %}
```

Comment:
Here you can set the path to `openshift_logging_mux_file_buffer_mount_path` with no if/else since we either specify the value or provide a default.